app/models/metric.rb

```diff
@@ -57,9 +58,11 @@ class Metric < ApplicationRecord
   #
   # +type+:: Which default metric to create. Currently only :hits are supported.
   def self.create_default!(type, attributes = {})
+    id = attributes[:service_id]
+    attributes.delete(:service_id)
```

Comment:
We're in `metric` so `id` would normally mean the Metric's ID. Better be explicit when declaring variables.
```suggestion
    service_id = attributes.delete(:service_id)
```